New version: XRT v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: XRT
- Repository: https://github.com/pc2/XRT.jl
- Created by: @DO6LTM
- Version: v0.2.0
- Commit: 9601b11a41b2630b4e4a97b1832c61e1c6141d23
- Git reference: v0.2.0
- Description: Julia wrapper around native XRT C++ API, a runtime for AI Engines and FPGA platforms
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- see the changelog
`````
<!-- END RELEASE NOTES -->

DO6LTM commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- Add switching between XRT versions
    - Add conditional compiling to xrtwrap.cpp
- Improve detection of native XRT installation
- Add Message APIs
- Add Configuration APIs
- Add Custom IP APIs
- Add Xclbin API data type
    - Add visualize_streams function
- Add device detection
    - Support Info APIs
    - Add XilinxDevice type
- Add tabular prints for devices and Xclbin files
- Add global state type
- Move enums to submodules
- Add directional BOArrays and wrapper types
- Import @prepare_bitstream macro
    - Add type annotations
    - Support direct cu execution
    - Support directional BOArrays
- Add @sync_buffers macro
- Add @prepare_run macro
- Add xbutil/xclbinutil support
- Reexport ArrayAllocators.jl
- Add unit- and integration tests
- Add benchmarks for C/C++/Rust/Python/Julia
- Update docs

JuliaHubRegistrator committed:
New version: XRT v0.2.0

UUID: 4d396880-2e3f-4be1-8c36-e87777010d00
Repo: https://github.com/pc2/XRT.jl.git
Tree: c9558a6c1a7e3f266c4f17b09ca4f9bd77b90dd3

Registrator tree SHA: d5716b7a540e5fbc43640c2fff2906fe50e9525a
